config/migrations/2024/20241112160000-remove-iv-from-provincies.sparql

```diff
@@ -0,0 +1,23 @@
+PREFIX besluit: <http://data.vlaanderen.be/ns/besluit#>
+PREFIX lmb: <http://lblod.data.gift/vocabularies/lmb/>
+
+DELETE {
+  GRAPH ?eenheidsGraph {
+    ?installatievergadering a lmb:Installatievergadering.
+    ?installatievergadering ?p ?o.
```

Comment:
?p and ?o not specified in the where part, will not be deleted

Reply:
Indeed 🤦🏻 

Ran it and removed 24 more triples